industry_benchmarks/input_structures/prepared_structures/merck/shp2/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,6 @@
+# GSK Industry Benchmark 
+Prepared by Alexander Williams
+## SHP2 Set
+## Software Used
+1. Maestro Protein Prep v.2024-02ß
+
```

Comment:
Same as above, additional information about protein and ligand prep options would be great for future reference.